docs/reference/cluster_manifest.md

```diff
@@ -173,6 +173,20 @@ These parameters are grouped directly under  the `spec` key in the manifest.
   [administrator docs](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#load-balancers-and-allowed-ip-ranges)
   for more information regarding default values and overwrite rules.
 
+* **masterServiceAnnotations**
+  A map of key value pairs that gets attached as [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+  to the master services created for the database cluster. Check the
+  [administrator docs](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#load-balancers-and-allowed-ip-ranges)
+  for more information regarding default values and overwrite rules.
+  This field overrides `serviceAnnotations` for the master service if not empty.
```

Comment:
```suggestion
  This field overrides `serviceAnnotations` with the same key for the master
  service if not empty.
```

Reply:
I updated the admin doc here: https://github.com/zalando/postgres-operator/pull/2161/commits/59016029914b694bb66b1e9db72de2bcc67f42be